### ISSUE TITLE
Dataset corresponding to which domain number

### ISSUE THREAD

**Issue by Tianananana** [xmed-lab/CPR#4]
Thanks for your great work. From the fundus datasets you provided, there are 4 domains, but in your paper, you only mentioned 
Drishti-GS, RIM-ONE, and REFUGE. Can I clarify which domain number corresponds to which dataset and also which dataset do the extra domain belongs to? Thank you.

**Comment by zheangh** [comment]
Thanks for your interest! The domain numbers in the code are accurate, so you can reproduce the results if you follow the Readme file.
The datasets were originally provided by https://github.com/emma-sjwang/BEAL. Specifically, Domain1 is Drishti-GS, Domain2 is RIM-ONE-r3, Domain3 is the training set of REFUGE, and Domain4 is the validation set of REFUGE.
You may want to get more details by looking into the REFUGE challenge summary paper "Refuge challenge: A unified framework for evaluating automated methods for glaucoma assessment from fundus photographs" or this paper "Patch-based Output Space Adversarial Learning for Joint Optic Disc and Cup Segmentation".
